ompi/mca/coll/libnbc/nbc_ireduce.c

```diff
@@ -479,11 +479,11 @@ static inline int red_sched_chain (int rank, int p, int root, const void *sendbu
   fragcount = count / numfrag;
 
   for (int fragnum = 0 ; fragnum < numfrag ; ++fragnum) {
-    offset = fragnum * fragcount * ext;
+    offset = fragnum * (size_t)fragcount * (MPI_Aint)ext;
```

Comment:
Is the `(MPI_Aint)` necessary here? I don't think so since the `size_t` cast on the LHS
```suggestion
    offset = fragnum * (size_t)fragcount * ext;
```